modules/environments/data.tf

```
@@ -0,0 +1,11 @@
+data "azuread_group" "acr" {
+  display_name = "DTS ACR Access Administrators"
+}
+
+data "azurerm_client_config" "current" {}
+
+data "azurerm_subscriptions" "service_endpoint_subscription" {
+  count               = var.pipeline_environment == "prod" ? 1 : 0
+  display_name_prefix = "DTS-SHAREDSERVICES"
```

a comment explaining this would be good.

```suggestion
  # Environment service connections are associated to the SDS subscriptions
  # because they need to be associated to at least one even if you are using a different one
  display_name_prefix = "DTS-SHAREDSERVICES"
```